_posts/2018-11-21-f18_03.md

```diff
@@ -0,0 +1,43 @@
+
+Goldberg’s opponent, LaBonge, was a member of the Los Angeles City Council from 2001 to 2015 representing the 4th district. Before that, when running for the City Council position in 1993, he narrowly lost to Goldberg by a four-percent margin of votes (Russell). Goldberg was a liberal, focusing her campaign towards union workers, teachers, feminists, LGBTQ people, and minorities. LaBonge, a strong conservative, focused his campaign toward people with a track record of voting in past elections. He mainly targeted those that were conservative. As shown in this artifact, Goldberg criticized LaBonge for accepting financial contributions from registered lobbyists, including the National Rifle Association’s local branch; however, LaBonge was actually in favor of gun control (Russell). By associating him with the NRA, Goldberg put LaBonge in a negative light, as gun control was a sensitive topic. Seventy percent of Americans were in favor of enforcing stricter gun laws (Gallup), as most people were still traumatized from the violence during LA Riots. As a result, Goldberg was the favored candidate at the time and her advertisement against LaBonge’s NRA endorsement portrayed her in a more positive light and helped secure her victory in the 1993 election.
+
+America’s gun control debate traces back to the eighteenth-century. In the twentieth-century, gun ownership and federal regulation of gun control became a major issue, with the National Rifle Association (NRA) as the focal point. The NRA was founded in 1871, initially as an organization to improve the marksmanship of soldiers. Following the implementation of legislation bringing state militia units under federal control in 1903 and creation of the position of the Director of Civilian Marksmanship in 1916, the NRA has been gradually recognized as one of the most influential special interest groups in the country (Marlin). Westside Firearms Association, the local branch of the NRA in Los Angeles in 1993, endorsed LaBonge’s group for the City Council. Having gained support from the NRA, this group opposed every legislation that repressed gun activity. For example, they opposed the ban on Saturday Night Specials, the handgun registration, and the waiting periods for gun purchases. Working against LaBonge and the NRA, Goldberg fought to ban Saturday Night Specials, which helped the later implementation of the Brady Handgun Violence Prevention Act in 1994. The act focused on decreasing gun violence by reducing the availability of handguns to criminals and other potentially dangerous people, thus leading to a 10-year federal ban on assault weapons (Monroe).
+
+![Jackie Goldberg Political Advertisement.](images/Goldberg1.png)
+{:.image}
+Political advertisement. Jackie Goldberg, 1993. Image courtesy of UCLA Special Collections.
```

Comment:
Your image captions need the formatting tag! The formatting should look like this:
https://gist.github.com/kirschbombe/824930bbe036dec9b40ff08943cb9500 
(Hint: you are missing the "caption" tag)